src/workloads/query/DensifyHours.yml

```diff
@@ -254,8 +272,11 @@ Actors:
               bounds: *bounds
             }
           }
+        }, {
+          $limit: *limit
```

Comment:
Just to confirm - $limit is here to emulate RunCommand behavior of reading only the first batch?

Reply:
yes